Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail.py

```diff
@@ -1464,6 +1481,51 @@ def send_email_command(client: MsGraphClient, args):
     return_outputs(human_readable, ec)
 
 
+def prepare_outputs_for_reply_mail_command(reply, email_to, message_id):
+    reply.pop('attachments', None)
+    to_recipients, cc_recipients, bcc_recipients = build_recipients_human_readable(reply)
+    reply['toRecipients'] = to_recipients
+    reply['ccRecipients'] = cc_recipients
+    reply['bccRecipients'] = bcc_recipients
+    reply['ID'] = message_id
+
+    message_content = assign_params(**reply)
+    human_readable = tableToMarkdown(f'Replied message was successfully sent to {", ".join(email_to)} .',
+                                     message_content)
+
+    return CommandResults(
+        outputs_prefix="MicrosoftGraph",
+        readable_output=human_readable,
+        outputs_key_field="SentMail",
+        outputs=message_content,
+    )
+
+
+def reply_email_command(client: MsGraphClient, args):
+    """
+    Reply to an email from user's mailbox, the sent message will appear in Sent Items folder
+    """
+    email_to = argToList(args.get('to'))
+    email_from = args.get('from', client._mailbox_to_fetch)
+    message_id = args.get('inReplyTo')
+    email_body = args.get('body', "")
+    email_subject = 'Re: ' + args.get('subject', "")
+    attach_ids = argToList(args.get('attachIDs'))
+    email_cc = argToList(args.get('cc'))
+    email_bcc = argToList(args.get('bcc'))
+    html_body = args.get('htmlBody')
+    attach_names = argToList(args.get('attachNames'))
+    attach_cids = argToList(args.get('attachCIDs'))
+    message_body = html_body if html_body else email_body
+
+    suffix_endpoint = f'/users/{email_from}/messages/{message_id}/reply'
+    reply = client.build_message_to_reply(email_to, email_cc, email_bcc, email_subject, message_body, attach_ids,
+                                          attach_names, attach_cids)
+    client.ms_client.http_request('POST', suffix_endpoint, json_data={'message': reply}, resp_type="text")
```

Comment:
What is the response here? Is there a possibility of a failed request with a non-failure error code?

Reply:
There is no useful response here, error handling of that request is done in MicrosoftApiModule (MicrosoftClient.http_request).